run_benchmarks.sh

```diff
@@ -14,16 +14,12 @@
 
 set -e
 
-RENAISSANCE_V="0.9.0"
-GRAALCE_V="1.0.0-rc16"
-J9_V="0.15.1"
+. `dirname $0`/paths.sh
```

Comment:
I wonder if we should put the following code before the `. dirname` line?

```sh
if [ ! -f `dirname $0`/paths.sh ]; then
    echo You need to run `make` before running this file 1>&2
    exit 1
fi
```

(or something like that).

Reply:
We can do, although if the user invoked the script via the makefile, the make deps will have ensured this.

I'd be inclined to add it anyway, as the user may have run the script manually.

Reply:
Yeah, I'm imagining people rushing to run the first sane sounding `.sh` script without reading the instructions.

Reply:
The backticks in your echo lead to hilarity :)

Reply:
Damn!